axlearn/cloud/gcp/measurement.py

```diff
@@ -109,47 +258,8 @@ def record(self, event: measurement.Event, *args, **kwargs):
                 event,
             )
 
-    def start_monitoring(self, *args, **kwargs):
-        """Starts Monitoring of Goodput.
-
-        Instantiate ml-goodput-measurement's GoodputMonitor to asynchronously calculate
-        Goodput and Badput at the upload_interval and upload to the specified TensorBoard
-        directory.
-        Note: This function requires initialization of distributed JAX before it is called.
-        If there are internal GCP errors from querying and uploading data, these will be
-        logged without affecting the workload. GoodputMonitor logs will provide further
-        information if data is not being uploaded correctly.
-
-        Default behavior is to push metrics to Google Cloud Monitoring.
-        This behavior can be overridden by configuring `goodput_monitoring.GCPOptions`
+    def start_monitoring(self, **kwargs):
```

Comment:
https://github.com/apple/axlearn/blob/a0144986dc6eb049107f99e4884f36cfeeb0cd87/axlearn/common/launch_trainer.py#L132

Since `start_monitoring` in GoodputRecorder is a dummy function in this PR,  and monitor initialization is done by `maybe_monitor_all`, but `maybe_monitor_all` is invoked in the `trainer.run` method,  we are calling

```
measurement.record_event(measurement.Event.START_JOB)
```
before monitor initialization.  This doesn't seem correct.

Can we fully revert back to original implementation, so we can get this functionality merged in, and worry about API changes in a standalone follow-up PR(s)?

Reply:
Thanks